sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Clauses/WhereClause.cs

```diff
@@ -8,20 +8,28 @@ namespace Azure.DigitalTwins.Core.QueryBuilder
     /// <summary>
     /// Custom object for a WHERE clause. Only meant to be used when adding WHERE to a query. Hidden from user.
     /// </summary>
-    internal class WhereClause : ClauseBase
+    internal class WhereClause
     {
         /// <summary>
-        /// Condition object that encodes the logical condition behind the WHERE clause.
+        /// Condition object represented in string format that encodes the logical condition behind the WHERE clause.
         /// </summary>
-        internal ConditionBase Condition { get; set; }
+        internal string Condition { get; set; }
 
         /// <summary>
-        /// Constructor for a WHERE clause.
+        /// Constructor for a WHERE clause using the internal condition object.
```

Comment:
You don't need to specify what it uses as input since you already have that as a parameter doc comment.